x-pack/legacy/plugins/siem/public/components/ml/create_descriptions_list.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiText, EuiSpacer, EuiFlexGroup, EuiFlexItem, EuiLink } from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+import { Anomaly, NarrowDateRange } from './types';
+import { getScoreString } from './get_score_string';
+import { PreferenceFormattedDate } from '../formatted_date';
+import { createInfluencers } from './create_influencers';
+import { DescriptionList } from '../../../common/utility_types';
+import * as i18n from './translations';
+import { createExplorerLink } from './create_explorer_link';
+
+const LargeScore = styled(EuiText)`
+  font-size: 45px;
+  font-weight: lighter;
+`;
+
+export const createDescriptionsList = (
```

Comment:
nit: s/createDescriptionsList/createDescriptionList

Reply:
Re-worded both the file name and all the places it is being used.